jsonclasses_pymongo/preload.py

```diff
@@ -63,6 +63,17 @@ def seedobject(cls: type[PObject], obj: dict[str, Any], oid: str | int, original
             frcls = field.foreign_class
             field_ref_name = cls.cdef.jconf.ref_name_strategy(field)
             result[field_ref_name] = getidref(frcls, getfieldvalue(obj, field))
+        elif field.fdef.fstore == FStore.FOREIGN_KEY and field.fdef._use_join_table:
+            ids = []
+            frcls = field.foreign_class
+            for id in getfieldvalue(obj, field):
+                ids.append(getidref(frcls, id))
+            field_ref_name = cls.cdef.jconf.ref_name_strategy(field)
+            add_ids = []
+            for id in ids:
+                add_ids.append({'_add': id})
+            result[field.name] = add_ids
```

Comment:
Does this really work? Is there any tests for this? I thought `result.set({field.name: add_ids})` may work